db/re/item_db_usable.yml

```diff
@@ -742,7 +741,7 @@ Body:
     Flags:
       BuyingStore: true
     Script: |
-      itemheal rand(44,66),0;
+      itemheal rand(22,33),0;
```

Comment:
It was fine according to aegis data and there is no value in kro description, please revert

Reply:
Thanks !